packages/node_modules/@webex/redux-module-spaces/src/actions.js

```diff
@@ -178,17 +183,22 @@ export function updateSpaceWithActivity(activity, isSelf, isReadable = false) {
 export function fetchSpace(sparkInstance, space) {
   return (dispatch) => {
     dispatch(storeInitialSpace(space.id));
+    dispatch(createConversationBegin());
 
     return sparkInstance.internal.conversation.get(space, {
-      activitiesLimit: 25,
+      activitiesLimit: 40,
       participantsLimit: -1,
       participantAckFilter: 'all',
-      includeParticipants: true
+      includeParticipants: true,
+      latestActivity: true
     })
       .then((fullSpace) => {
         dispatch(storeUsers(fullSpace.participants.items));
         dispatch(storeActivities(fullSpace.activities.items));
         dispatch(storeSpaces([fullSpace]));
+        dispatch(storeConversation(
```

Comment:
Same note here about adding a comment.

Reply:
I've added a variation of this comment to help explain these details. Let me know if this is okay.